### ISSUE TITLE
Documentation - FE technology/decision

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#21]
## Acceptance criteria
* Section in Wiki to describe FE technology decision making
* Small listing of discussed technologies

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Lucas-Nan** [comment]
not only technology but more all kinds of decisions made in fe

**Comment by Lucas-Nan** [comment]
https://github.com/amosproj/amos2024ss04-building-information-enhancer/wiki/Frontend-Decisions
